.github/workflows/codeql-analysis.yml

```diff
@@ -1,8 +1,10 @@
 name: CodeQL Analysis
 
 on:
-  pull_request:
   push:
+    branches:
+      - development
+  # pull_request:  ## temporarily disable for pull requests until https://github.com/ubiquity/ubiquity-dollar/issues/876 is resolved.
 
 jobs:
   code-ql-analysis:
```

Comment:
Why not disable it entirely including on push? You mentioned that Code QL analysis is likely to be an issue

Reply:
all right let's disable completely for now